code/ietf-sid-file.yang

```diff
@@ -174,6 +174,27 @@ module ietf-sid-file {
         other things.";
     }
 
+    leaf sid-file-status {
+      type enumeration {
+         enum unpublished {
+             description
+                "This SID file is unpublished [RFC8407], also called
+                a work-in-progress or workfile.
+                This may be when it accompanies an unpublished YANG
+                module, or when only the SID file itself is unpublished.
+                The 'item' list MAY contain entries with a status
+                value of 'unstable'.";
+         }
+         enum published {
+             description
+                "This SID file is published, for a published YANG
+                module. The 'item' list MUST NOT contain entries with
+                a status value of 'unstable'.";
+         }
+      }
+      default published;
+    }
+
     list dependency-revision {
```

Comment:
It is OK with me to leave this global field out.
The status leaf is sufficient to remove the DISCUSS

IMO this might warn humans right at the start of the file.
The start (header) should be useful to humans with the 'more' command

Reply:
It is optional...  (And need not be present in published modules.)

To help the `more` crowd, I moved up the leaf right after `sid-file-version`: 0f4bae0